### ISSUE TITLE
Crash on code completion

### ISSUE THREAD

**Issue by JoelSylvesterDukosiAccount** [rationalmatter/Juno-Issue-Tracker#428]
**Describe the bug**
App crash when code completion attempts to find completions for letter 's', when an imported library starts with 's'. It's sensitive to the imported library. The example below crashes, but a simple 'import scipy' does not. I've not seen the behaviour with any other library or initial letter. App does not crash if code completion is disabled.

**To Reproduce**
Try this example

```
### reproduce Juno crash on code completion of 's'

### only reproduced for this import
from scipy.signal import StateSpace

#with code completion on, start typing 's...'

def main():
    print('import from scipy')
    
if __name__ == "__main__":
    main()

```
**Expected behavior**
To not crash!

**Screenshots**

https://github.com/user-attachments/assets/d9283bdf-58d0-44cd-9c16-6532f0404915

**Environment (please complete the following information):**
 - Device: iPad Pro (12.9-inch) (5th generation)
 - iOS version: iPadOS 17.6.1
 - Juno 3.3.0

**Comment by alexstaravoitau** [comment]
Thanks for reporting @JoelSylvesterDukosiAccount—looks like a crash in scipy when Juno is trying to access/import one of its submodules. I will take a look!

**Comment by alexstaravoitau** [comment]
This should be fixed in Juno 3.4.0, which is now live on the App Store.
